src/main/java/com/contentful/java/cma/model/CMALocale.java

```diff
@@ -182,11 +185,49 @@ public CMALocale setDefault(boolean isDefault) {
   }
 
   /**
+   * @return true if this locale is used on the content management api.
+   */
+  public boolean isContentManagementApi() {
+    return contentManagementApi;
+  }
+
+  /**
+   * Change this locale to be the  used on the management api.
+   *
+   * @param contentManagementApi to update the value with.
+   * @return this instance for ease of chaining.
+   */
+  public CMALocale setContentManagementApi(boolean contentManagementApi) {
+    this.contentManagementApi = contentManagementApi;
+    return this;
+  }
+
+  /**
+   * @return true if this locale is used on the content delivery api.
+   */
+  public boolean isContentDeliveryApi() {
+    return contentDeliveryApi;
+  }
+
+  /**
+   * Change this locale to be available in the content deliver api.
+   *
+   * @param contentDeliveryApi true if available.
+   * @return this instance for ease of chaining.
+   */
+  public CMALocale setContentDeliveryApi(boolean contentDeliveryApi) {
+    this.contentDeliveryApi = contentDeliveryApi;
+    return this;
+  }
+
+  /**
    * @return a human readable string, representing the object.
    */
   @Override public String toString() {
     return "CMALocale { " + super.toString() + " "
         + "code = " + getCode() + ", "
+        + "contentDeliveryApi = " + isContentDeliveryApi() + ", "
+        + "contentManagementApi = " + isContentManagementApi() + ", "
```

Comment:
Are boolean automatically cast to string, in Java? 🙂

Reply:
Yes, almost. Actually they are first boxed to `Boolean` and then it's `.toString()` is called ... ;)